src/tests/BuildWasmApps/Wasm.Build.Tests/PInvokeTableGeneratorTests.cs

```diff
@@ -115,6 +115,43 @@ public static int Main()
             Assert.Contains("Main running", output);
         }
 
+        [Theory]
+        [BuildAndRun(host: RunHost.Chrome)]
+        public void UnmanagedStructsAreConsideredBlittable(BuildArgs buildArgs, RunHost host, string id)
+        {
+            string code = 
+            """
+            using System;
+            using System.Runtime.CompilerServices;
+            using System.Runtime.InteropServices;
+
+            [assembly: DisableRuntimeMarshalling]
+
+            public class Test
+            {
+                public static int Main()
+                {
+                    var x = new S { Value = 5 };
+            
+                    Console.WriteLine("Main running " + x.Value);
```

Comment:
This is using the value from the local struct. Can we call `M(..)`?

Reply:
err.. get a callback?

Reply:
The error comes from the build. I'm using the struct it just don't get trimmed.

Reply:
It would test whether we generated a good signature for it, right?

Reply:
Yes, when it succeeds it doesn't produce any warning.